usagov/vote-gov-drupal
# Feature/vote 2874 downsync workflow improvements

## Jira ticket

[VOTE-2874](https://cm-jira.usa.gov/browse/VOTE-2874)

## Description

Implement improvements to downsync workflow.

## Deployment and testing

### QA/Testing instructions

1. Allow the scheduled backup to run in the production environment.
2. Confirm a `latest.sql.gz` is created in the top level of the prod directory in the backup S3 bucket.
3. Run the downsync-backup workflow to create a manual backup of the prod database by manually triggering a pipeline with `key/value: backup/prod` as the pipeline parameters. This will need to be ran when the upkeep workflow is not running as either a pipeline or cloud.gov task.
5. Confirm a `latest.sql.gz` has been created in the top level of the prod directory in the backup S3 bucket with an updated timestamp from the previous one.
6. Run the manually triggered downsync-restore workflow in a branch specifying `key/value: restore/<env name>` as the pipeline parameters. For example, `restore/stage` for restoring the latest prod db backup into the stage env.
7. Confirm workflow completes successfully and the stage env is backed up (for this example).

## Checklist for the Developer




## Checklist for the Peer Reviewers
